djnro/local_settings.py.dist

```diff
@@ -135,6 +135,33 @@ REALM_COUNTRIES = (
     ('country_2letters', 'Country' ),
 )
 
+# List the login methods to be offered to users here.
+# The fields to list for each method are:
+#   backend: backend name.  This has to match the backend name in the
+#            corresponding python-social-auth class, or be one of:
+#            'shibboleth', 'locallogin'
+#   enabled: True or False.  Should this login method be offered to users?
+#   class:   Backend class to load.  Gets added to
+#            settings.AUTHENTICATION_BACKENDS automatically for enabled login methods.
+#   name:    Human readable name of the authentiation method to present to users
+#
+#   local_image: Relative path of a local static image to use as logo for the login method.
+#   image_url:   Full URL of an image to use as logo for the login method.
+#   fa_style:    Font-Awesome style to use as logo for the login method.
+#
+# The first four elements (backend, enabled, class, name) are REQUIRED.
+# One of the logo elements (local_image, image_url, fa_style) SHOULD also be provided.
+MANAGE_LOGIN_METHODS = (
+  { 'backend': 'shibboleth', 'enabled': True, 'class': 'djangobackends.shibauthBackend.shibauthBackend', 'name': 'Shibboleth', 'local_image': 'img/image_shibboleth_logo_color.png' },
+  { 'backend': 'google-oauth2', 'enabled': True, 'class': 'social.backends.google.GoogleOAuth2', 'name': 'Google', 'fa_style': 'fa fa-google fa-2x' },
+  { 'backend': 'twitter', 'enabled': False, 'class': 'social.backends.twitter.TwitterOAuth', 'name': 'Twitter', 'fa_style': 'fa fa-twitter fa-2x' },
+  # add more here
+)
+
+# Add enabled backends to authentication backends
+EXTRA_AUTHENTICATION_BACKENDS += tuple([ m['class'] for m in  MANAGE_LOGIN_METHODS if m['enabled'] and m['class'] ])
```

Comment:
This amends settings.AUTHENTICATION_BACKENDS. Even though the order therein does not matter in our case (not sure about duplicates though), because we don't have a single URL where multiple/all backends are chained for authentication, it would be good to at least disable or remove from that setting backends that may be enabled through MANAGE_LOGIN_METHODS.